receiver/jmxreceiver/integration_test.go

```diff
@@ -43,25 +43,33 @@ import (
 
 type JMXIntegrationSuite struct {
 	suite.Suite
-	JARPath string
+	VersionToJar map[string]string
 }
 
 func TestJMXIntegration(t *testing.T) {
 	suite.Run(t, new(JMXIntegrationSuite))
 }
 
 func (suite *JMXIntegrationSuite) SetupSuite() {
-	jarPath, err := downloadJMXMetricGathererJAR()
-	require.NoError(suite.T(), err)
-	suite.JARPath = jarPath
+	jmxJarReleases := map[string]string{
```

Comment:
Can you move these up to a top level constant so they're most obvious for someone updating after a java-contrib release?

Reply:
Fixed